build-system/status-page/comment.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Add progress comment for Stable and LTS cherry-picks
+ */
+
+const fetch = require('node-fetch');
+const [number, body, user] = process.argv.slice(2);
```

Comment:
Almost all other parts of `build-system` destructure the result of `require('minimist')(process.argv.slice(2))` to extract args. Worth doing here too?

Reply:
Good point but we won't be using flags for args. Sneak peek on how these scripts will be called - https://github.com/estherkim/good-bot/blob/main/.github/workflows/status-page.yml#L26

If these scripts were called by a developer I'd see the benefits of using `minimist` and adding flags but since it will be called by a GitHub Action with a static order, it shouldn't really matter.

(re: consistency, the args in the bento publishing scripts are destructure in the same way)

Reply:
All these points SGTM. Carry on!